test/spec/formmodel.spec.js

```diff
@@ -1721,10 +1721,11 @@ describe('merging an instance into the model', () => {
     describe('when a deprecatedID node is not present in the form format', () => {
```

Comment:
Several of these tests fail if run with the new `modelStr` and `instanceStr` but without the new code in `form-model.js`.

Reply:
Sorry for interrupting and to do so late but I happened to see that we already have `instanceIdEl`, so `instanceIdEl.after(deprecatedIdEl)` would be even nicer.